crates/testing/src/slt/runner.rs

```diff
@@ -46,19 +47,7 @@ impl SltRunner {
 
     pub fn test_files_dir(mut self, dir: impl AsRef<Path>) -> Result<Self> {
         let dir = dir.as_ref();
-
-        if !dir.exists() {
-            return Err(anyhow!("Path `{}` does not exist", dir.to_string_lossy()));
-        }
-
-        if !dir.is_dir() {
-            return Err(anyhow!(
-                "Path `{}` is not a valid directory",
-                dir.to_string_lossy()
-            ));
-        }
-
-        // Get the absolute path
+        ensure_dir(dir)?;
```

Comment:
this one looks like it's not supposed to create the dirs. We can probably revert this.

Reply:
Thanks for the catch. I've updated the commit 🚀